### PR TITLE
add ward page

### LINKED ISSUE #105: Ward overview
```[tasklist]
### Ward overview
- [ ] #104 
```

Tasks - overview of wards in an organization-2

### PR DESCRIPTION
**Pull request by DasProffi**
closes #105

### CODE REVIEW

**Review by use-to — CHANGES_REQUESTED** [review]
move `pages/wards.tsx` to `pages/organizations/[uuid].tsx`

File: tasks/components/cards/WardCard.tsx [hunk 1]
```diff
@@ -42,18 +50,22 @@ export const WardCard = ({
...
   return (
     <Card onTileClick={onTileClick} isSelected={isSelected} className={tw('group cursor-pointer')}>
       <div className={tw('flex flex-row justify-between w-full')}>
         <span className={tw('font-bold font-space')}>{ward.name}</span>
         <button onClick={onEditClick}
                 className={tw('hidden group-hover:block')}>{translation.edit}</button>
       </div>
-      <div className={tw('text-left my-1')}>{ward.roomNames.join(', ')}</div>
+      <div
+        className={tx('text-left my-1', { 'text-gray-400': !hasRooms })}>{hasRooms ? ward.rooms.map(value => value.name).join(', ') : translation.noRoomsYet}</div>
```
Comment (use-to) [inline tasks/components/cards/WardCard.tsx:64]: this is a bit too distracting for my taste
maybe make it a little bit smaller and bleach the color a bit

File: tasks/components/layout/WardDisplay.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      <ColumnTitle title={translation.wards}/>
+      <div className={tw('grid grid-cols-2 gap-6')}>
+        {wards.map(ward => (
+          <WardCard
+            key={ward.id}
+            ward={ward}
+            isSelected={selectedWard?.id === ward.id}
+            onEditClick={() => onSelectionChange(ward)}
+            onTileClick={() => {
+              // TODO open room view for wards
```
Comment (use-to) [inline tasks/components/layout/WardDisplay.tsx:64]: You can already implement the routing to `/ward/<id>` here.

**Review by use-to — CHANGES_REQUESTED** [review]
This pr looks really promising

File: tasks/components/layout/WardDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+type WardDTO = {
+  id: string,
+  name: string,
+  rooms: Room[],
+  unscheduled: number,
+  inProgress: number,
+  done: number
+}
```
Comment (use-to) [inline tasks/components/layout/WardDetails.tsx:63]: I saw this concept once in a while...
Can't we just use the DTO defined in the mutations? Whats the reason for this?

Comment (DasProffi) [reply tasks/components/layout/WardDetails.tsx:63]: You mean the multiple type definitions, right? It's to make the components more independent, thereby increasing the reusability. It also allows each component to only require the subset of attribute it needs. We can change that or discuss it, I don't mind importing the definitions

Comment (use-to) [reply tasks/components/layout/WardDetails.tsx:63]: Okay, that's resolved. I'm facing this in #161
So I'll probably fix it there.

**Review by use-to — APPROVED** [review]
Good job! Looking forward to review the next page!

**Pull request merged**